sql/api/src/main/scala/org/apache/spark/sql/errors/ExecutionErrors.scala

```diff
@@ -238,6 +238,15 @@ private[sql] trait ExecutionErrors extends DataTypeErrorsBase {
         "encoderType" -> encoder.getClass.getName,
         "docroot" -> SparkBuildInfo.spark_doc_root))
   }
+
+  def zoneOffsetError(e: java.time.DateTimeException): SparkDateTimeException = {
+    new SparkDateTimeException(
+      errorClass = "INVALID_TIMEZONE",
+      messageParameters = Map.empty,
```

Comment:
Provide the bad input value.

Reply:
Now we provide the bad input value for error message. Thanks!!